src/health-tm/health-tm-lambda.js

```diff
@@ -0,0 +1,82 @@
+const fetch = require('node-fetch')
+
+const TM_STATUS_URL =
+  'https://tasking-manager-tm4-production-api.hotosm.org/api/v2/system/heartbeat/'
+
+const TM_STATISTICS_URL =
+  'https://tasking-manager-tm4-production-api.hotosm.org/api/v2/system/statistics/?abbreviated=true'
+
+const parseEvent = (event) => {
+  const snsJSON = JSON.stringify(event.Records[0].Sns)
+  const snsObject = JSON.parse(snsJSON)
+  const decodedSns = JSON.parse(decodeURIComponent(snsObject.Message))
+
+  return decodedSns
+}
+
+const createBlock = (status, mappersOnline, totalProjects) => {
+  return [
+    {
+      type: 'section',
+      text: {
+        type: 'mrkdwn',
+        text:
+          status === 'healthy'
+            ? ':white_check_mark: *The Tasking Manager is operational*'
+            : ':heavy_exclamation_mark: *The Tasking Manager cannot be reached*',
+      },
+    },
+    {
+      type: 'section',
+      fields: [
+        {
+          type: 'mrkdwn',
+          text: `:female-construction-worker::male-construction-worker:*Number of Mappers Online*: ${mappersOnline}`,
+        },
+        {
+          type: 'mrkdwn',
+          text: `:world_map: *Number of Projects Hosted*: ${totalProjects}`,
+        },
+      ],
+    },
```

Comment:
Nice emoji usage. :raised_hands: :slightly_smiling_face: -- this is the nice part about slack bots!